Customizing/global/lang/ilias_de.lang.local

```diff
@@ -1334,7 +1334,7 @@ gev#:#load_signature_list#:#Unterschriftenliste herunterladen
 gev#:#crs_load_signature_list#:#Unterschriftenliste herunterladen
 gev#:#load_member_list#:#Mitgliederliste abrufen
 gev#:#crs_load_member_list#:#Mitgliederliste abrufen
-gev#:#view_maillog#:#Maillog einsehen
+gev#:#view_maillog#:#Mailing einsehen
```

Comment:
Die Sprachvariable ist gleichzeitig der key für den OP in der rbac_op. Wenn wir die Variable hier ändern muss das in der Datenbank auch nachgezogen werden. Trotzdem machen?